atd-vze/src/Components/GridTable.js

```diff
@@ -142,7 +143,7 @@ const GridTable = ({
         setChartQuery(updatedChartsQuery);
       }
     }
-  }, []); // eslint-disable-line react-hooks/exhaustive-deps
+  }, [query.gql]); // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
We should probably remove the linter rule exception

Reply:
it is needed. It expects a longer list of dependencies thats unnecessary imho.